data/sources/categories.json

```diff
@@ -460,7 +476,11 @@
         "Season of the Worthy",
         "Season of the Hunt",
         "Artifacts - IGNORE",
-        "Ornaments - IGNORE"
+        "Ornaments - IGNORE",
+        "Source: Acquired from Ada-1",
+        "Source: Splicer Servitor",
```

Comment:
not sure i understand why ignoring Ada-1 and Splicer Servitor. are we specifically aiming for only sources that support the Inventory page?

Reply:
that was my intent here as these are just mods

Reply:
cool looks good